evap/evaluation/migrations/0114_remove_username.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.0.5 on 2020-05-23 11:50
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('evaluation', '0113_import_names'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='userprofile',
+            options={'ordering': ('last_name', 'first_name', 'email'), 'verbose_name': 'user', 'verbose_name_plural': 'users'},
+        ),
+        migrations.RemoveField(
+            model_name='userprofile',
+            name='username',
```

Comment:
fwiw, this is probably not reversible. (since we never checked, we probably already have other migrations that are not reversible, and i know we didn't need a reversible migration history for a while, so maybe we shouldn't care).

Reply:
true, as the username was not nullable this migration is not reversible. in case we should run in serious troubles after the update it shouldn't be too hard to manually reverse the migration by filling the usernames with the email addresses.
but as you said, we most probably won't need that at all. so should i add some additional code for making it reversible or shall we just leave it as it is?